notebook/static/notebook/js/shortcuteditor.js

```diff
@@ -116,7 +116,10 @@ var KeyBindingList = createClass({
             "Casing will have no effects: (e.g: `;` and `:` are the same on english keyboards)."+
             " You need to explicitelty write the `Shift` modifier.\n"+
             "Valid modifiers are `Cmd`, `Ctrl`, `Alt` ,`Meta`, `Cmdtrl`. Refer to developper docs "+
-            "for their signification depending on the platform."
+            "for their signification depending on the platform."+
```

Comment:
'signification' isn't really a word; I'm not sure what you're trying to say here?

Also, on the line above, 'developer' has just one p.